dev/progress.md

```diff
@@ -0,0 +1,56 @@
+# Progress
```

Comment:
I don't like this file. IMO this should live in issues / boards.

Reply:
Sounds good! I'll look more into the GH tools and keep this for myself.